Muehle_Algo.ipynb

```diff
@@ -75,7 +80,10 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "`value_minimax(State, player)`"
+    "Die Funktion `value_minimax(State, player)` erhält drei Argumente. Einen Spielzustand, einen Spieler und die Suchtiefe. Die Funktion gibt dabei den Wert zurück, den der übergebene Spielzustand für den übergeben Spieler hat.\n",
+    "Dieser Wert wird, für den Fall, dass das Spiel mit dem übergebenen Zustand beendet ist, von der Funktion `finished()` berechnet. Wenn die maximale Suchtiefe erreicht wurde wird der Wert allerdings von der Funktion `heuristic()` berechnet.\n",
+    "Ist die maximale Suchtiefe noch nicht erreicht wird rekursiv nach dem besten Folgezustand gesucht.\n",
```

Comment:
Das würde ich noch genauer erklären.
Was passiert hier:
`max([ -value_minimax(to_tuple(ns), o, depth) for ns in next_states(to_list(state), player) `

Warum max, warum über den Wert des Gegners, warum negativ?

Reply:
Da du vom Gegner aus suchst musst du den Wert für dich umdrehen

Reply:
Für die nextstates ist ja der immer der Gegner am Zug